ember.md

```diff
@@ -359,10 +361,22 @@ export default DS.Model.extend({
   firstName: attr('string'),
   lastName: attr('string'),
 
-  // Associations
+  // Relationships
   children: hasMany('child'),
-
-  // Computed Properties
+  
+  // Properties
+  PeopleAgeBrackets: [
```

Comment:
Please change this example to have a lower-case `p` to match our normal style for attributes (`peopleAgeBrackets`).